presto-docs/src/main/sphinx/connector/iceberg.rst

```diff
@@ -435,10 +440,31 @@ Create an Iceberg table partitioned by the first letter of the team field::
         partitioning = ARRAY['truncate(team, 1)']
     );
 
-.. note::
+Create an Iceberg table partitioned by ``ds``::
```

Comment:
No change to this actual line! 

I can't add a comment on line 426/431 ("Create an Iceberg table partitioned by the first letter of the team field") so I'm adding it here. 
Can you revise that line to format "the team field" to "the ``team`` field"?